src/web/mjs/connectors/SmackJeeves.mjs

```diff
@@ -1,130 +1,99 @@
 import Connector from '../engine/Connector.mjs';
 
-/**
- *
- */
 export default class SmackJeeves extends Connector {
 
-    /**
-     *
-     */
     constructor() {
         super();
         super.id = 'smackjeeves';
         super.label = 'Smack Jeeves';
         this.tags = [ 'webtoon', 'english' ];
-        this.url = 'http://www.smackjeeves.com';
+        this.url = 'https://www.smackjeeves.com';
+        this.he = require('he');
```

Comment:
`he` should be available as global (on window), if `require` is used something seems not to be as intended.
`require` is only for installed electron packages.